docs/docs/guides/integrate/login/oidc/webkeys.md

```diff
@@ -0,0 +1,361 @@
+---
+title: OpenID Connect and Oauth2 web keys
+sidebar_label: Web keys
+---
+
+Web Keys in ZITADEL are used to sign and verify JSON Web Tokens (JWT).
+ID tokens are created, signed and returned by ZITADEL when a OpenID connect (OIDC) or Oauth2
+authorization flow completes and a user is authenticated.
+Optionally zitadel can return JWTs for access tokens if the OIDC Application is configured for it.
+
+## Introduction
+
+ZITADEL uses asymmetric cryptography to sign and validate JWTs.
+Keys are generated in pairs resulting in a private and public key.
+Private keys are used to sign tokens.
+Public keys are used to verify tokens.
+OIDC clients need the public key to verify ID tokens.
+Oauth2 API apps might need the public key if they want to client-side verification of a
+JWT access tokens, instead of [introspection](/docs/apis/openidoauth/endpoints#introspection_endpoint).
+ZITADEL uses public key verification when API calls are made or when the user-info or introspection
```

Comment:
```suggestion
OAuth2 API apps might need the public key if they want to client-side verification of a
JWT access tokens, instead of [introspection](/docs/apis/openidoauth/endpoints#introspection_endpoint).
ZITADEL uses public key verification when API calls are made or when the userInfo or introspection
```